File: Linguagem C++/Estrutura Enquanto/MediaIdades.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{
    int idades,soma,cont;
    double media;

    cout << "Digite as idades:" << endl;
    cin >> idades;

    if (idades <=0){
        cout << "IMPOSSIVEL CALCULAR" << endl;
    }else {
        soma = 0;
        cont = 0;


    while (idades > 0){
        soma = soma + idades;
        cont++;
        cin >> idades;
     };


    media = (double)soma / cont;
    cout << fixed << setprecision(2);
    cout << "MEDIA = " << media << endl;

    };


    return 0;
}

File: Linguagem C#/Enquanto/MediaIdades.cs
using System;
using System.Globalization;

namespace MediaIdades {
    class Program {
        static void Main(string[] args) {

            CultureInfo CI = CultureInfo.InvariantCulture;

            int idade,cont,soma;
            double media;

            Console.WriteLine("Digite as idades: ");
            idade = int.Parse(Console.ReadLine());

            cont = 0;
            soma = 0;
            if (idade <= 0) {
                Console.WriteLine("IMPOSSIVEL CALCULAR");
            }else {
                while (idade > 0) {
                    cont++;
                    soma = soma + idade;
                    idade = int.Parse(Console.ReadLine());
                };
                media = (double)soma / cont;
                Console.WriteLine("MEDIA = "+media.ToString("F2",CI));
            };
        }
    }
}
